tests/unit/lib/cli_validation/test_image_repository_validation.py

```diff
@@ -90,12 +87,31 @@ def test_image_repository_validation_success_IMAGE_image_repositories(
 
     @patch("samcli.lib.cli_validation.image_repository_validation.click")
     @patch("samcli.lib.cli_validation.image_repository_validation._is_all_image_funcs_provided")
-    @patch("samcli.lib.cli_validation.image_repository_validation.get_template_artifacts_format")
+    def test_image_repository_validation_success_IMAGE_image_repository_with_no_all_image_func_provided(
+        self, is_all_image_funcs_provided_mock, mock_click
+    ):
+
+        is_all_image_funcs_provided_mock.return_value = False
+        mock_context = MagicMock()
+        mock_context.params.get.side_effect = [
+            False,
+            False,
+            "123456789012.dkr.ecr.us-east-1.amazonaws.com/test1",
+            False,
+            False,
+            None,
+            MagicMock(),
+        ]
+        mock_click.get_current_context.return_value = mock_context
+        self.foobar()
+
+    @patch("samcli.lib.cli_validation.image_repository_validation.click")
```

Comment:
Duplicate test as the one on line 49 (`test_image_repository_validation_success_IMAGE_image_repository`) of this file now that the refactor happened?

Reply:
Good call, the change was already included in that line 49 test; must have been a left over of my first PR version.
Edited and rebased.